documentation/installation.md

```diff
@@ -48,6 +47,12 @@ Execute the bootstrap script to begin the installation process:
 ```sh
 ./scripts/bootstrap.sh
 ```
+The script defaults to using the central repository at `(https://github.com/redhat-ai-services/ai-accelerator.git)` and the main branch for GitOps configuration. If your working remote repository differs, the script will prompt you to update the GitOps configuration to match. Selecting this option will:
+
+- Update the repository and branch in `cluster-config-app-of-apps`.
+- Commit the changes locally.
+- Require you to push the updated file `cluster-config-app-of-apps` to the central - repository `(https://github.com/redhat-ai-services/ai-accelerator.git)` and the main branch before submitting a pull request.
```

Comment:
Was trying to think like. a new user seeing this for the first time, and the third step might be a little confusing since you'd only need to perform that when creating a pull request, not for running bootstrap.sh. What do you think about wording such as:

The GitOps configuration used by this bootstrap script contains default parameters that poinjt to the main project repository found at `(https://github.com/redhat-ai-services/ai-accelerator.git)` using the `main` branch. If you have created a different branch, or even forked this repository to your own remote repository, the script detect this and will prompt you to update your GitOps configuration to match. Selecting this option will:

- Update the repository and branch in `cluster-config-app-of-apps`.
- Commit the changes locally.

If you have changes you would like to submit back to the project in the form of a pull request, you must revert your repo configuration updates inside `cluster-config-app-of-apps` so that they point to the base project (`https://github.com/redhat-ai-services/ai-accelerator.git`) and main branch before submitting.